tests/chainer_tests/functions_tests/loss_tests/test_crf1d.py

```diff
@@ -0,0 +1,89 @@
+import unittest
+
+import numpy
+
+import chainer
+from chainer import functions
+from chainer import gradient_check
+from chainer import testing
+
+
+class TestCRF1d(unittest.TestCase):
```

Comment:
How about adding new tests regarding a toy-example that can analytically compute the maximum of the CRF model and its state?